### PR TITLE
Adds a new credit song if the station was nuked, adds support for context-sensitive credits

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
I'll need @I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE to tell me if moving on_round_end further down the line breaks anything
I'll need @d3athrow to add a bunch of songs to the media repo

:cl:
 * rscadd: Added a new credits song, it will play if the station was nuked.

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
The PR will be delayed until I can grab some MP3 files for:
- A shortened version of Blow It Up where it kicks into the vocals a few seconds in
- Twin Peaks
- You Are The Blood
- Nuclear

**Comment by boy2mantwicethefam** [comment]
I've decided to forego the shortened version of Blow It Up for just the whole thing
Added a new variable that admins can toggle for credits if they want certain credits songs instead

File: code/modules/credits/credits.dm [hunk 1]
```diff
@@ -126,6 +127,8 @@ var/global/datum/credits/end_credits = new
  */
 /datum/credits/proc/on_round_end()
 	draft()
+	if(!do_not_change_credits_song)
```
Comment (ShiftyRail) [inline code/modules/credits/credits.dm]: Double negative in this context is confusing, make it a `change_credits_song` and invert the boolean checks

File: code/modules/credits/credits.dm [hunk 2]
```diff
@@ -262,6 +265,22 @@ var/global/datum/credits/end_credits = new
 		cast_string += "<h3>[true_story_bro]</h3><br>In memory of those that did not make it.<br>[english_list(corpses)].<br>"
 	cast_string += "</div><br>"
 
+//This proc will run at round-end and run various conditions to change audio_link
+//Currently only hosts one additional song
+/datum/credits/proc/determine_round_end_song()
+	if(do_not_change_credits_song) //How did this happen?
```
Comment (ShiftyRail) [inline code/modules/credits/credits.dm]: Useless code, remove.

File: code/modules/credits/credits.dm [hunk 3]
```diff
@@ -262,6 +265,22 @@ var/global/datum/credits/end_credits = new
...
+		return
+	var/list/candidates
```
Comment (ShiftyRail) [inline code/modules/credits/credits.dm]: Initialise this as a list. You cannot add things to a `null` variable.

**Comment by d3athrow** [comment]
took barry almost 2 months to get me the songs how i requested them.

**Comment by Kammerjunk** [comment]
2 months and counting

**Pull request merged**